.gitignore

```diff
@@ -68,5 +68,9 @@ bin/
 # Gradle files
 .gradle/
 
+# node files
+node/
+node_modules/
```

Comment:
Yes, but how sure are we that we won't want to check in a directory called `node` at some point somewhere in zanata-server?  `node` directories may not always represent output files.